app/src/main/res/xml/preferences.xml

```diff
@@ -238,11 +280,6 @@
                 android:title="@string/pref_name_autoSyncQueue_enabled"
                 android:summary="@string/pref_desc_autoSyncQueue_enabled"
                 android:defaultValue="false"/>
-            <CheckBoxPreference
-                android:key="@string/pref_key_autoDlNew_enabled"
```

Comment:
Yeah, the result of "service.addArticle(url)" is now added right away.

Reply:
I now realize that this is not the same as before (because new articles added from other clients won't appear until sync), but I think it covers the main idea: articles added from current client appear when they are added.